packages/paste-website/src/pages/components/alert-dialog/index.mdx

```diff
@@ -0,0 +1,209 @@
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Alert Dialog"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/alert-dialog"
+  storybookUrl="/?path=/story/components-alert-dialog--alert-dialog-with-two-actions"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<Callout>
+  <CalloutTitle>Using alpha components in your product</CalloutTitle>
+  <CalloutText>
+    Alpha components are ready for use in production and components like Alert Dialog won't likely see any API changes.
+    The term "Alpha" means there is only basic documentation at this time. Please share your experience using this Alpha
+    component by{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions/new" showExternal>
+      starting a discussion on GitHub
+    </Anchor>
+    .
+  </CalloutText>
+</Callout>
+
+## Guidelines
+
+### About Alert Dialog
+
+The Alert Dialog should be used when a user flow needs to be stopped unless a specific action is taken. It is used to convey
+important messages that can't be ignored like [Alert](/components/alert) content. Potential use-cases for Alert Dialog are:
+
+- Confirming a permanent change, like deleting user-made data
+- Relaying an important system message like a maintenance downtime window
+
+While visually similar to the [Modal](/components/modal), the Alert Dialog ensures the user acknowledges the information in the dialog.
+Users cannot interact with content outside an active Alert Dialog window until the user explicitly closes it.
+When the Alert Dialog opens, focus moves to an element contained in it (by default, the first element that can be focused),
+and the focus should stay in and cycle through the Alert Dialog's content. Focus shouldn't return
+to the underlying page until the user closes the Alert Dialog. This can happen in any of the following ways:
+
+- Presses a "Cancel" button in the Alert Dialog footer
+- Presses a "Submit" or "Confirm" button in the Alert Dialog footer
+
+### Accessibility
+
+- All elements required to interact with the Alert Dialog, including closing or acknowledging it, are contained in the
+  Alert Dialog since they trap focus, and users can't interact with the underlying page.
+- Tabbing through a Alert Dialog will cycle through its content in the same way it does on a page.
+- The element that serves as the Alert Dialog container has a role of `alertdialog`.
+- The Alert Dialog is labelled by the `heading` prop.
+
+## Examples
+
+### Default Alert Dialog
+
+<LivePreview scope={{AlertDialog, Button}} noInline>
+  {defaultExample}
+</LivePreview>
+
+### Destructive Alert Dialog
+
+<LivePreview scope={{AlertDialog, Button}} noInline>
+  {destructiveExample}
+</LivePreview>
+
```

Comment:
Can you add an example of an alert dialog being opened on top of a modal, like this story? https://5e53448165911c0022e68c74-lgvaybmuxb.chromatic.com/?path=/story/components-alert-dialog--open-alert-dialog-from-modal

Reply:
Sure thing. Example has been added.